www/src/theme.js

```diff
@@ -24,33 +24,30 @@ const colors = {
   darker: '#20232a', // really dark blue
   brand: '#61dafb', // electric blue
   brandLight: '#bbeffd',
+  brandOnWhite: '#10819b',
   text: '#1a1a1a', // very dark grey / black substitute
-  subtle: '#777', // light grey for text
+  subtle: '#6d6d6d', // light grey for text
+  subtleOnDark: '#999',
   divider: '#ececec', // very light grey
   note: '#ffe564', // yellow
   error: '#ff6464', // yellow
   white: '#ffffff',
   black: '#000000',
 };
 
-const fonts = {
-  header: {
-    fontSize: 60,
-    lineHeight: '65px',
-    fontWeight: 700,
-  },
-  small: {
-    fontSize: 14,
-  },
-};
-
 const SIZES = {
   xsmall: {min: 0, max: 599},
   small: {min: 600, max: 739},
   medium: {min: 740, max: 979},
   large: {min: 980, max: 1279},
   xlarge: {min: 1280, max: 1339},
   xxlarge: {min: 1340, max: Infinity},
+
+  // Tweakpoints
+  xlargeSmaller: {min: 1100, max: 1339},
+  belowSidebarFixed: {min: 740, max: 1559}, // Used for "between()"
+  sidebarFixed: {min: 1560, max: Infinity},
+  sidebarFixedNarrowFooter: {min: 1560, max: 2000},
```

Comment:
I don't think these constants make sense, (paricularly _xlargeSmaller_ 😁 ), with the way the sizes were designed. If we _need_ these new sizes, then I think we should pick a few hard-coded media query values rather than try to make them work within the `media` methods.

Reply:
See my latest commit. I've not removed them entirely, as I think it's good to have all width constraints in one central place, but I've made it more obvious about what they're actually doing. And was able to remove `belowSidebarFixed` due to the new "excludeLarge" on the between function 👍